custom-plugins/fictive_studios/admin/templates/listing/partials/template-list-helper.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+if(!class_exists('WP_List_Table')){
+    require_once( ABSPATH . 'wp-admin/includes/screen.php' );
+    require_once( ABSPATH . 'wp-admin/includes/class-wp-list-table.php' );
+}
+
+class Template_List_Table extends \WP_List_Table
+{
+
+    public function __construct()
+    {
+        parent::__construct(
+            array(
+                'singular' => 'Draft',
+                'plural' => 'Drafts',
+                'ajax' => false,
+            )
+        );
+    }
+
+    protected function get_brocha_templates($search_term = null)
+    {
+        global $wpdb;
+        $table_name = $wpdb->prefix . 'brocha_templates';
+        $query = "SELECT * FROM $table_name";
+        if ($search_term) {
+            $query = $wpdb->prepare("SELECT * FROM $table_name WHERE name LIKE %s", '%' . $wpdb->esc_like($search_term) . '%');
+        } else {
+            $query = "SELECT * FROM $table_name";
+        }
+        $results = $wpdb->get_results($query, ARRAY_A);
```

Comment:
What does ARRAY_A represent?

Reply:
will be fixed when the base class is ready